apps/api/src/server.py

```diff
@@ -263,73 +274,132 @@ def get_buy_txn(token_id: UUID, user_address: str, amount_sei: int) -> dict:
 @app.post("/runtimes")
 def create_runtime(background_tasks: BackgroundTasks) -> Runtime:
     """
-    Creates a new runtime via github actions.
-    Runtime will not truly be started until the github action is completed, and aws resources are up and running (~5min)
-    Raises a 500 if the github action fails to start.
-    Returns the runtime object.
+    Attempts to create a new runtime.
+    Returns a Runtime object immediately, with flag started=False.
+    The runtime will be polled for 10 min to verify that it is online.
+    If it is not, then the runtime will be deleted.
     """
+    # Perhaps, the least performant code ever written
+    # This is why *real* companies use leetcode
     with Session() as session:
-        runtimes = crud.get_runtimes(session)
-        runtime_count = len(runtimes)
-    next_runtime_number = runtime_count + 1
-    if os.getenv("ENV") == "staging":
-        url = f"https://aiden-runtime-{next_runtime_number}.staigen.space"
-        actions_url = "https://api.github.com/repos/abstractoperators/aiden/actions/workflows/145628373/dispatches"
-    elif os.getenv("ENV") == "prod":
-        url = f"https://aiden-runtime-{next_runtime_number}.aiden.space"
-        actions_url = "https://api.github.com/repos/abstractoperators/aiden/actions/workflows/144070661/dispatches"
-    else:
-        # TODO: dev env w/ local runtime.
-        pass
-
-    # Store the runtime in the database
-
-    GITHUB_WORKFLOW_DISPATCH_PAT = os.getenv("GITHUB_WORKFLOW_DISPATCH_PAT")
-    next_runtime_number = runtime_count + 1
-    try:
-        resp = requests.post(
-            actions_url,
-            headers={
-                "Accept": "application/vnd.github+json",
-                "Authorization": f"Bearer {GITHUB_WORKFLOW_DISPATCH_PAT}",
-                "X-GitHub-Api-Version": "2022-11-28",
-            },
-            json={
-                "ref": "main",
-                "inputs": {
-                    "service-no": str(next_runtime_number),
-                },
-            },
-            timeout=3,
+        runtimes = crud.get_runtimes(
```

Comment:
this is demonic. are we planning on multiple agents per runtime in the future? or how are we going to distribute agents across runtimes